test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -4481,6 +4481,7 @@ public void PackCommand_PackLicense_SimpleExpression_StandardLicense(string lice
                     var licenseMetadata = nuspecReader.GetLicenseMetadata();
                     Assert.Equal(new Uri(string.Format(LicenseMetadata.LicenseServiceLinkTemplate, licenseExpr)), new Uri(nuspecReader.GetLicenseUrl()));
                     Assert.NotNull(licenseMetadata);
+                    Assert.Equal(licenseMetadata.LicenseUrl.AbsoluteUri, nuspecReader.GetLicenseUrl());
```

Comment:
Tests should have the expected data verbatim, rather than as a result of some operation, because that operation itself might be wrong.